builtin/providers/aws/cloudfront_distribution_configuration_structure_test.go

```diff
@@ -44,6 +49,35 @@ func trustedSignersConf() []interface{} {
 	return []interface{}{"1234567890EX", "1234567891EX"}
 }
 
+func lambdaSetHash(v interface{}) int {
```

Comment:
Nah it's not your fault. 

@apparentlymart was correct in that the manually added `Set` function should be omitted, unless there's a really good reason. 

The problem here was reconstructing "sets" in _code_ in the test instead of configuration. All the previous examples I found in the codebase were simple set's of `strings` which is trivial to reconstruct, but `NewSet` is more difficult with sets that have a more complicated schema. 

Like I said you did 99% of the work here I just needed to do some battlefield surgery 😄